applications/nrf5340_audio/src/bluetooth/bt_renderer/bt_volume_control.c

```diff
@@ -0,0 +1,370 @@
+		int ret;
+
+		for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
+			if (vcs_client_peer[i] != NULL) {
+				ret = bt_vcp_vol_ctlr_unmute_vol_up(vcs_client_peer[i]);
+				if (ret) {
+					LOG_WRN("Failed to volume up for remote channel %d, ret = "
+						"%d",
+						i, ret);
+				}
+			}
+		}
+
+		return 0;
+	} else if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
+		return bt_vcp_vol_rend_unmute_vol_up();
+	}
+
+	LOG_WRN("VCP not enabled");
+	return -EIO;
+}
+
+int bt_vol_ctrl_down(void)
+{
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		int ret;
+
+		for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
+			if (vcs_client_peer[i] != NULL) {
+				ret = bt_vcp_vol_ctlr_unmute_vol_down(vcs_client_peer[i]);
+				if (ret) {
+					LOG_WRN("Failed to volume down for remote channel %d, ret "
+						"= %d",
+						i, ret);
+				}
+			}
+
+			return 0;
+		}
+	} else if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
+		return bt_vcp_vol_rend_unmute_vol_down();
+	}
+
+	LOG_WRN("VCP not enabled");
+	return -EIO;
+}
+
+int bt_vol_ctrl_mute(void)
+{
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		int ret;
+
+		for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
+			if (vcs_client_peer[i] != NULL) {
+				ret = bt_vcp_vol_ctlr_mute(vcs_client_peer[i]);
+				if (ret) {
+					LOG_WRN("Failed to mute for remote channel %d, ret "
+						"= %d",
+						i, ret);
+				}
+			}
+		}
+
+		return 0;
+	} else if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
+		return bt_vcp_vol_rend_mute();
+	}
+
+	LOG_WRN("VCP not enabled");
+	return -EIO;
+}
+
+int bt_vol_ctrl_unmute(void)
+{
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		int ret;
+
+		for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
+			if (vcs_client_peer[i] != NULL) {
+				ret = bt_vcp_vol_ctlr_unmute(vcs_client_peer[i]);
+				if (ret) {
+					LOG_WRN("Failed to unmute for remote channel %d, "
+						"ret = %d",
+						i, ret);
+				}
+			}
+		}
+
+		return 0;
+	} else if (IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
+		return bt_vcp_vol_rend_unmute();
+	}
+
+	LOG_WRN("VCP not enabled");
+	return -EIO;
+}
+
+int bt_vol_ctrl_vcs_discover(struct bt_conn *conn)
+{
+	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+
+		int ret, index;
+
+		if (vcs_client_peer_exists(conn)) {
+			return -EAGAIN;
+		}
+
+		index = vcs_client_peer_index_free_get();
+		if (index < 0) {
+			return index;
+		}
+
+		ret = bt_vcp_vol_ctlr_discover(conn, &vcs_client_peer[index]);
+		return ret;
+	}
+
+	LOG_ERR("VCP volume controller not enabled");
+	return -ECANCELED;
+}
+
+int bt_vol_ctrl_vcs_client_init(void)
+{
+	if (!IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
+		LOG_ERR("VCP volume controller not enabled");
+		return -ECANCELED;
+	}
+
+	static struct bt_vcp_vol_ctlr_cb vcs_client_callback;
+
+	vcs_client_callback.discover = vcs_discover_cb_handler;
+	vcs_client_callback.state = vcs_state_ctlr_cb_handler;
+	vcs_client_callback.flags = vcs_flags_ctlr_cb_handler;
+	return bt_vcp_vol_ctlr_cb_register(&vcs_client_callback);
+}
+
+int bt_vol_ctrl_vcs_server_init(void)
```

Comment:
server <-> rendrer and client <-> controller. Can we consider renaming our functions? E.g. 
```suggestion
int bt_vol_ctrl_vcs_rend_init(void)
```

Reply:
Yeah, we can do that